cli/command/system/version.go

```diff
@@ -48,16 +52,22 @@ Server:{{if ne .Platform.Name ""}} {{.Platform.Name}}{{end}}
    {{- end}}
   {{- end}}
  {{- end}}
-{{- end}}{{end}}`
+ {{- end}}{{- end}}
+ {{- if .KubernetesOK}}{{with .Kubernetes}}
+ Orchestrator:
+   Kubernetes:	{{.Kubernetes}}
```

Comment:
Minor naming nit: I assume in the future we will need to change "Server" to "Engine".

Maybe this should be formatted as:

```
Kubernetes:
  Version: {{.Kubernetes}}
  Stack API: {{.StackAPI}}
```

I think that would be more consistent with the other sections. What do you think?

Reply:
My first thought was to generalize this section to all orchestrators, but finally it seems that we don't have much things to print from swarm, and also this section seems very specific to Kubernetes. So I'm with renaming it.